website/src/pages/index.js

```diff
@@ -95,7 +95,7 @@ function Index() {
                 <Link href="/install" title="Use GitHub Application">
                   Install Application
                 </Link>
-                <Link href="/action" title="Use GitHub Action">
+                <Link href="/docs/products/action" title="Use GitHub Action">
                   Use Action
                 </Link>
```

Comment:
```suggestion
                <a href="/docs/products/action" title="Use GitHub Action">
                  Use Action
                </a>
```

Nice! It doesn't work because I used `Link` component and should have used `a` instead. Same with the "Install Application" link. Could you update those two? :))